crash_similarity.py

```diff
@@ -83,14 +83,10 @@ def delete_old_models(current_date, force_train):
     delete all models in case of user forces new training,
     if not, delete all models except of today's model
     """
-    models = [f for f in os.listdir('./models')]
     if force_train:
-        for model in models:
-            os.remove('./models/' + model)
+        [os.remove('./models/' + model) for model in os.listdir('./models')]
     else:
-        for model in models:
-            if current_date not in model:
-                os.remove('./models/' + model)
+        [os.remove('./models/' + model) for model in os.listdir('./models') if current_date not in model]
```

Comment:
@ibrahimsharaf There are still two places where `os.remove` is called. Fix it, please.